http/src/main/resources/reference.conf

```diff
@@ -640,7 +640,7 @@ image {
   welderGcrUri = "us.gcr.io/broad-dsp-gcr-public/welder-server"
   welderDockerHubUri = "broadinstitute/welder-server"
   welderHash = "6648f5c"
-  jupyterImage =  "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.2.8"
+  jupyterImage =  "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.3.0"
```

Comment:
Ohhh good catch, i don't think that modifying this file is mentioned in the documentation :(